### ISSUE TITLE
Sales-Order Processing- Credit Invoice with deposit (CM). CM does not appear in GST reports. Please check all GST reports. Thanks

### ISSUE THREAD

**Issue by KatrinaCagaanan** [a2000-erp-team/WEBERP#433]
![image.png](https://images.zenhubusercontent.com/5cf8a04f2e4fe4691d7f073e/34dc0d34-d839-45db-b8db-3e6e8e04c8d9)
![image.png](https://images.zenhubusercontent.com/5cf8a04f2e4fe4691d7f073e/62b01e59-521e-4393-8bc7-68f178f15ba9)

**Comment by KatrinaCagaanan** [comment]
Hi @ZawZawThein, will need to re-open this. Kindly do for all GST reports. Thanks.
GST Grouping listing is ok, but other GST reports are not ok:
![image.png](https://images.zenhubusercontent.com/5cf8a04f2e4fe4691d7f073e/1a56d7f1-8b72-4e2a-9498-311d46e3db98)
GST Posting is still blank:
![image.png](https://images.zenhubusercontent.com/5cf8a04f2e4fe4691d7f073e/cca8a0d7-b484-4ec1-b308-4acd324ea915)
GST Form 5 and GST Form 5 by GST Type:
![image.png](https://images.zenhubusercontent.com/5cf8a04f2e4fe4691d7f073e/80c83b30-5734-4dcb-b1ac-56823d598f18)
GST IN OUT:
![image.png](https://images.zenhubusercontent.com/5cf8a04f2e4fe4691d7f073e/b2384843-3bd0-468a-b7f3-2c4faefa422a)
![image.png](https://images.zenhubusercontent.com/5cf8a04f2e4fe4691d7f073e/adf1357a-7ff6-479d-9b16-47f77cac3d9a)

**Comment by KatrinaCagaanan** [comment]
Hi @ZawZawThein ,

Yesterday GST Grouping Listing was ok already. But today when I double check report is empty. I created CE,CR,CA and CM that should appear in the report.

![image.png](https://images.zenhubusercontent.com/5cf8a04f2e4fe4691d7f073e/bd14f9fe-9b22-4cb5-bb4e-4eda3fb27206)

These are the transactions I created.

![image.png](https://images.zenhubusercontent.com/5cf8a04f2e4fe4691d7f073e/2ba18947-014b-4579-a342-8ffbad540e32)

Btw, I create a separate ticket per GST report from now on so you will not get confused.:) Sorry if this ticket has a lot of reports to fixed.
